delivery/routes.py

```diff
@@ -34,3 +37,27 @@ def put(self, restaurant_id: int):
         restaurant.id = restaurant_id
         updated_restaurant = self.restaurants.update(restaurant)
         return jsonify(self.schema.dump(updated_restaurant)), 200
+
+
+class LoginEndpoint(MethodView):
+    def __init__(self, users, schema):
+        self.users = users
+        self.schema = schema
+
+    def post(self):
+        login = request.json.get('login', None)
+        password = base64.b64encode(request.json.get('password', None).encode()).decode()
```

Comment:
hm... could you please explain what is happening here?

as far as I know, it should be like: `password` is sent as it is (via HTTPS), but `hashed_password` is stored in DB, so `hash_password(password)` is compared with `hashed_password`

Reply:
may help here:
https://github.com/pallets/werkzeug/blob/master/src/werkzeug/security.py
https://medium.com/python-in-plain-english/implementing-flask-login-with-hash-password-888731c88a99

Reply:
changed